OMAF-Sample/README.md

```diff
@@ -72,4 +73,4 @@
 | cachePath | Cache path | /tmp/cache |
 
 
-   - **Note** : So far, some parameters settings are limited. URL need to be a remote dash source URL, example : `https://172.18.0.2:5000`. The parameter sourceType must set to 0, which represents dash source. The parameter decoderType must set to 0, which stands for FFmpeg software decoder. The parameter contextType need to be 0, which represents glfw context. And useDMABuffer flag should be set to 0. 
+   - **Note** : So far, some parameters settings are limited. URL need to be a remote dash source URL, choose `./run.sh 8K Live` for example : `https://172.18.0.2:5000/LIVE8K/Test.mpd`. The parameter sourceType must set to 0, which represents dash source. The parameter decoderType must set to 0, which stands for FFmpeg software decoder. The parameter contextType need to be 0, which represents glfw context. And useDMABuffer flag should be set to 0. 
```

Comment:
./run.sh 8K LIVE should be ./run.sh 8K LIVE

Reply:
Done